packages/demoreact/src/app/SpaceUITest/SpaceWidget.tsx

```diff
@@ -1,7 +1,155 @@
+import { useState } from "react";
+import { Section, SectionItem } from "../components/StyledComponents";
+import SpaceUITest from "./SpaceUITest";
```

Comment:
order of imports

Reply:
can ignore since this is demoreact app

Reply:
yeah. since it's only a testing app, let's not follow too strict standards there